src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -4957,6 +5018,23 @@ subroutine parse_G5DAS_met_tag( met_path_in, met_tag_in, date_time, &
 
        end if
 
+    elseif (met_tag_out(1:18)=='cross_d5924_GEOSIT') then
```

Comment:
@gmao-qliu : for consistency with RPIT/FPIT, I think we need to keep the GEOS version in the "cross" stream definition. As was the case with RPIT/FPIT, a future version of GEOS-IT might be produced with a newer GEOS system but still under the "GEOS-IT" name.  This change requires you do edit the MET_TAG in your LDAS.rc file.